v1/pipeline/steps/custom/harness-approval-step-info.yaml

```diff
@@ -4,39 +4,20 @@ allOf:
 - type: object
   required:
   - approvers
-  - includePipelineExecutionHistory
+  - include_pipeline_execution_history
   properties:
-    approvalMessage:
+    message:
       type: string
       minLength: 1
-    approverInputs:
+    inputs:
       type: array
       items:
         $ref: approver-input-info.yaml
     approvers:
       $ref: approvers.yaml
-    includePipelineExecutionHistory:
+    include_pipeline_execution_history:
```

Comment:
should we rename this to include_execution_history ?

Reply:
done